### ISSUE TITLE
Frame sequential/HD3D possible?

### ISSUE THREAD

**Issue by DieKatzchen** [BlueSkyDefender/Depth3D#10]
Dunno if it's at all possible, but my projector is an older 3d projector that only accepts a 120hz frame sequential signal over VGA for 3d. I think there's also some kind of signal to send it into 3d mode, dunno how that works. Most games that have native 3D I select AMD HD3D and it just works, so that's probably a broader target to hit, if it's even possible. Tridef also orks through HD3D.  Bino works too, but just calls it "left/right alternating" if that helps any.

**Comment by BlueSkyDefender** [comment]
Let me do some research on the subject. I will get back to you later.

**Comment by DieKatzchen** [comment]
Any progress on this?

**Comment by VR-Nima** [comment]
@BlueSkyDefender Do you have the link to the HD3D code?

**Comment by BlueSkyDefender** [comment]
Here you go.
Here is a PDF that explains some stuff.
http://www.mediafire.com/file/d40hn5gtqq4s3z1/Active_Quad_Buffer_Stereo_on_DirectX__10_and_11.pdf/file
And this link for the SDK
http://www.mediafire.com/file/11gf468iv772qaq/AMD_Quad_Buffer_SDK_v11.zip/file

**Comment by MopheusDG** [comment]
This is an old issue, probably never gonna be done, right ? I'd love to use my 3D projector with games but only supports frame-sequential and couldn't make it work any way.... tridef, nvidia, none of them.

**Comment by bo3b** [comment]
BlueSkyDefender suggested I might post this here.  This is an experimental build of Reshade that will generate a 3D Vision Direct mode output, which means that 3D Vision FrameSequential will work with SuperDepth3D_VR+.

http://bo3b.net/ReShade64.dll

Rename that to dxgi.dll or whatever is necessary for your DX11 game.  Only supports DX11 at the moment, and 3D Vision must be enabled, but it will operate in Direct Mode, not Automatic, so you can use SuperDepth3D_VR+ to generate the 3D.

Setup will be a drag, I'm sorry.  This is using a build based off Reshade 4.9.1, and that version of reshade doesn't work at all on the obvious driver of 425.31.  It hangs in a loading shaders loop.  Next logical driver is 452.06, last one working for 3D Vision.  You can use this driver if you use the global hack in 3DFM, or the BringBack3DV tool. And probably the special 3Dmigoto that ships with 3DFM/HelixVision will also work, but is untested.  


Experimental, just tested as functional in a single game, Rime, using 3D Vision monitor.  Please let me know if you get it working with 3D Vision hardware, including projectors that do frame sequential.

Give it a try if you are motivated, and let us know your experience here.  I'll monitor this issue and provide help where I can.

**Comment by DieKatzchen** [comment]
I have since upgraded my projector to one that does SBS and OU, but I'm pretty sure it can still do frame sequential so I'll try it this weekend.

**Comment by bo3b** [comment]
> I have since upgraded my projector to one that does SBS and OU, but I'm pretty sure it can still do frame sequential so I'll try it this weekend.

Sounds good.  Please let us know what you find.  

On a bunch of games, we've been getting black in the left eye, instead of stereo. UE4 games would work, but others not.  I've worked out the problem, and BlueSkyDefender has a fix in the works.  

In the meantime, fetch his latest shader from the repo, and change the lines 730 from:

` #if __RESHADE__ >= 40400 `

to 

```
#if` BUFFER_COLOR_BIT_DEPTH == 10 //This PreProcessor is not a bool it really is 8 or 10 not a bool
```

No change to the dxgi.dll I posted.

**Comment by AkitoFire** [comment]
I couldn't get it to work on my machine yet. I feel like I'm missing something here because I just get a standard SBS image in-game.
I own an official 3D Vision monitor and glasses kit, have the driver version 452.06 installed through 3DFM, have the global hack switch on, set up Reshade, and replaced the dxgi.dll with your file (The Reshade menu says that it is version 4.9.0 UNOFFICIAL), replaced the lines of code in SuperDepth3D_VR+ and toggled it on in Reshade's menu. I tested Nioh 2 and RiME but both just show a simple SBS image.
However, thank you for your time and effort! I hope I get it to work, I never imagined something like this would be developed.

**Comment by bo3b** [comment]
@AkitoFire: Yes- sorry forgot this key detail.  You need to also enable the HelixVision=1 flag in the settings while it's running.  Otherwise it's in normal SuperDepth3D mode.  Preprocessor defines at bottom of SuperDepth3D settings.

We are planning to bring all the setup into 3DFM/HelixVision to make it a one-click setup once we work out all the details.

**Comment by shinra358** [comment]
any progress? id also test things if need be. projector here also.

**Comment by bo3b** [comment]
Some progress, works now to convert a SuperDepth3D image into frame sequential 3D Vision, but only for DX11 games.

See this post for beta download and how to install:  https://steamcommunity.com/app/1127310/discussions/0/3076495389981189251/?ctp=4#c3127164056428324766

**Comment by DonFlymoor** [comment]
Would it be possible to use this with a custom ir emmiter and active 3D glasses? This repo has the code for the ir emmiter.
https://github.com/DonFlymoor/3DVisionAVR

**Comment by saimonGX1** [comment]
any progress for sequential frame 3d display ?

**Comment by BenGost** [comment]
> Some progress, works now to convert a SuperDepth3D image into frame sequential 3D Vision, but only for DX11 games.
> 
> See this post for beta download and how to install: https://steamcommunity.com/app/1127310/discussions/0/3076495389981189251/?ctp=4#c3127164056428324766

hi, does it work with the latest version of the driver? with a 3XXX or 4XXX rtx?
